src/Workspaces/Core/Portable/Workspace/Solution/ProjectDependencyGraph.cs

```diff
@@ -56,9 +56,11 @@ public partial class ProjectDependencyGraph
         private ImmutableDictionary<ProjectId, ImmutableHashSet<ProjectId>> _transitiveReferencesMap;
         private ImmutableDictionary<ProjectId, ImmutableHashSet<ProjectId>> _reverseTransitiveReferencesMap;
 
-        // Intentionally created with a null reverseReferencesMap. Doing so indicates _lazyReverseReferencesMap
-        //  shouldn't be calculated until reverse reference information is requested. Once this information
-        //  has been calculated, forks of this PDG will calculate their new reverse references in a non-lazy fashion.
+        /// <remarks>
+        ///   Intentionally created with a null reverseReferencesMap. Doing so indicates _lazyReverseReferencesMap
```

Comment:
📝 Doc comment is fine, but at that point you'd generally want `<see>` for references to actual code elements (`_lazyReverseReferencesMap` in this case)